bundles/org.openhab.binding.freeathomesystem/README.md

```diff
@@ -0,0 +1,150 @@
+# OpenHAB 3 ABB/Busch-free@home Smart Home binding
+
+ OpenHAB ABB/Busch-free@home binding based on the offical free@home local api
+
+![alt text](https://github.com/jannodeluxe/jannnnoooo/blob/main/free_at_home_logo_1.jpg)
+![alt text](https://github.com/jannodeluxe/jannnnoooo/blob/main/abb_freeathome_2_0.png)
+# Description
+
+This openhab binding allows you to connect your free@home Smart Home system from ABB / Busch-Jaeger to OpenHAB and to control and observe most of the components.
+It requires a System Access Point with version 2.6.1 or higher.
+
+# Features
+
+* Control your ABB/Busch-free@home system with its Lights, Outlets, Blinds, etc. from OpenHAB via local API.
+
```

Comment:
Remove all of this as it is already mentioned and is obvious. The supported devices goes into it in more detail.